proxy/proxy.go

```diff
@@ -244,6 +244,11 @@ type Params struct {
 	// It allows to add additional logic (for example tracing) by providing a wrapper function
 	// which accepts original skipper http.RoundTripper as an argument and returns a wrapped roundtripper
 	CustomHttpRoundTripperWrap func(http.RoundTripper) http.RoundTripper
+
+	// Registry provides key-value API which uses host from normalizeSchemeHost as a key
+	// and returns some metadata about endpoint which could be used to perform better load balancing,
+	// fadeIn, etc.
```

Comment:
I guess you mean that key is "host:port" string.
Also omit "which could be used to perform better load balancing, fadeIn, etc.".
User can see details about available data in the Metrics type docs.